code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -333,7 +331,7 @@
 	return
 
 /datum/species/proc/handle_pre_change(var/mob/living/carbon/human/H)
-	if(!H.get_species() == "Monkey" || !H.get_species() == "Farwa" || !H.get_species() == "Stok" || !H.get_species() == "Wolpin" || !H.get_species() == "Neara")
+	if(H.butcher_results)//clear it out so we don't butcher a actual human.
```

Comment:
I mean it'd be better if you moved this to the `handle_pre_change` of the monkey species but whatever